gke/terraform/gke.tf

```
@@ -39,7 +39,7 @@ module "gke" {
       name            = "base"
       machine_type    = "e2-medium"
       min_count       = 1
-      max_count       = 5
+      max_count       = 10
```

```suggestion
      max_count       = 7
```
10 es mucho, 7 esta bien